src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Repository/FilterMetaRepository.cs

```diff
@@ -53,17 +54,18 @@ public async Task CreateFilterMetas(
         IReadOnlySet<string> allowedColumns,
         CancellationToken cancellationToken = default)
     {
+        var publicIdMappings = await CreatePublicIdMappings(dataSetVersion, cancellationToken);
```

Comment:
Aha cool.  I see the original problem now - shame we didn't include mapping filters and indicators in MVP cos we would have spotted this by now otherwise!

Reply:
Yep indeed! A bit of a blind spot unfortunately. Will hopefully be sorting more of the tests out in a follow up ticket as well!